images/bootstrap/Dockerfile

```diff
@@ -124,4 +124,12 @@ COPY ["barnacle/barnacle", "/usr/local/bin/"]
 COPY ["entrypoint.sh", "runner.sh", "create_bazel_cache_rcs.sh", \
         "/usr/local/bin/"]
 
+# TODO(krzyzacy): Move the scenario scripts to kubekins v2
+# The bundled scenarios are for podutil jobs, bootstrap jobs will still use
+# scenario scripts from cloned test-infra
+RUN cp ../../scenarios ./scenarios
+RUN mkdir /workspace/scenarios
+COPY ["./scenarios", "/workspace/scenarios"]
```

Comment:
Why not just the following?
`COPY ["../../scenarios", "/workspace/scenarios"]`

Reply:
/shrug 
let me try... I remember you have to put things in the same dir as the Dockerfile...

Reply:
it does not have to be the same directory as the dockerfile, but it must be within the build context directory. ideally we should stop building things in tree and stage things we want to copy into the image within a temp dir or something.

Reply:
moved to Makefile